packages/server/src/utils/KeyvLRU.ts

```diff
@@ -0,0 +1,79 @@
+import LRUCache from 'lru-cache';
+import Keyv, { Store } from 'keyv';
+
+// LRUCache wrapper to implement the Keyv `Store` interface.
+export class LRUStore<T> implements Store<T> {
```

Comment:
Maybe call it LRUCacheStore, since it is specifically the store for `lru-cache`. (After all there already is a https://www.npmjs.com/package/keyv-lru)

Reply:
[15c5326](https://github.com/apollographql/apollo-server/pull/6154/commits/15c53268953f77f9967997e90a931eeb8beca7a4)